### DISCUSSION TITLE
Looking for a help

### DISCUSSION THREAD

**Discussion by schukau** [community/community#122790] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone!
I'm a composer and looking for someone who will help me to write programm for algorhythmic composition (e.g. Xenakis' Fortran app. Actions will be with midi track. I will give you a written algorhythm, so I need somebody to write this algorhythm as code. Python or Supercollider.

**Comment by ebndev** [comment]
Hi @schukau, we’re glad you’re here 😀

Thanks for sharing your idea with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. 

Thank you for helping us maintain a productive and tidy community for all our members.
